src/main/java/com/reshiftsecurity/plugins/intellij/gui/editor/BugsLineMarkerProvider.java

```diff
@@ -191,24 +192,25 @@ public String fun(final PsiElement psiElement) {
 		@SuppressWarnings({"HardcodedFileSeparator"})
 		private static String getTooltipText(final List<ExtendedProblemDescriptor> problemDescriptors) {
 			final StringBuilder buffer = new StringBuilder();
+			EducationCachingService _eduCacheService = ServiceManager.getService(EducationCachingService.class);
 			buffer.append("<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\">");
-			buffer.append("<HTML><HEAD><TITLE>");
-
-			final int problemDescriptorsSize = problemDescriptors.size();
-			for (int i = 0; i < problemDescriptorsSize; i++) {
-				final ExtendedProblemDescriptor problemDescriptor = problemDescriptors.get(i);
-				buffer.append("");
-				buffer.append("</TITLE></HEAD><BODY><H3>");
-				buffer.append(BugInstanceUtil.getBugPatternShortDescription(problemDescriptor.getBug().getInstance()));
-				buffer.append("</H3>");
-				buffer.append(PATTERN.matcher(BugInstanceUtil.getDetailText(problemDescriptor.getBug().getInstance())).replaceAll(""));
-				if (i < problemDescriptors.size() - 1) {
-					buffer.append("<HR>");
-				}
-
-			}
-
-			buffer.append("</BODY></HTML>");
+			buffer.append(_eduCacheService.getBriefOverview(problemDescriptors, false));
```

Comment:
is this protected against xss?

Reply:
should be addressed once sanitization is added to `getBriefOverview`